ocs_ci/ocs/constants.py

```diff
@@ -1484,15 +1484,15 @@
 }
 
 PERFORMANCE_PROFILE_REQUEST_CPU_VALUES = {
-    "mgr": "1500m",
-    "mon": "1500m",
+    "mgr": "1500m'",
```

Comment:
I think it is typo here - "1500m'", added '-symbol

Reply:
Addressed